docs/index.md

```diff
@@ -101,6 +101,7 @@
   - [iso8601ToRippleTime](#iso8601torippletime)
   - [rippleTimeToISO8601](#rippletimetoiso8601)
   - [txFlags](#txflags)
+  - [txFlagIndices](#txflagindices)
```

Comment:
I know this was the pre-existing name, but `txFlagIndices` sounds weird to me. Naming is hard, but ... could we put some thought into a better name for this?

Reply:
If were willing to change this, I think we should call them `AccountSetFlags` or `AccountFlags`